airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1353,6 +1366,14 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/ImportError'
+    
+    Metadatabase:
+      type: object
+      description: Returns the status of the metadatabase
+      properties:
+        statuses:
+          type: string
```

Comment:
What values are allowed here? Is it enum?

Reply:
Here is the change `b3ec805`

For now I have used these enums for status:
- `healthy` 
- `unhealthy`